releasenotes/notes/gRPC-status-10ee5819a4faf151.yaml

```diff
@@ -0,0 +1,3 @@
+upgrade:
+  - |
+    Aggregate APM stats payloads by gRPC code
```

Comment:
I think the notes need to be changed? This isn't an "upgrade" per se relative to the main change we already merged.